src/components/AttachmentCarousel/index.js

```diff
@@ -29,6 +32,9 @@ const propTypes = {
     /** Callback to update the parent modal's state with a source and name from the attachments array */
     onNavigate: PropTypes.func,
 
+    /** Callback to change the download button Visibility */
+    toggleDownloadButtonVisibility: PropTypes.func,
```

Comment:
```suggestion
    setDownloadButtonVisibility: PropTypes.func,
```

We're not toggling the visibility. We're setting it based on the value passed as param.